packages/flutter/lib/src/physics/spring_simulation.dart

```diff
@@ -12,36 +12,36 @@ import 'utils.dart';
 ///
 /// Used to configure a [SpringSimulation].
 class SpringDescription {
-  /// Creates a spring given the mass, spring constant and the damping coefficient.
+  /// Creates a spring given the mass, stiffness, and the damping coefficient.
   ///
-  /// See [mass], [springConstant], and [damping] for the units of the arguments.
+  /// See [mass], [stiffness], and [damping] for the units of the arguments.
   const SpringDescription({
     this.mass,
-    this.springConstant,
+    this.stiffness,
     this.damping
   });
 
-  /// Creates a spring given the mass (m), spring constant (k), and damping
-  /// ratio (ζ). The damping ratio is especially useful trying to determing the
-  /// type of spring to create. A ratio of 1.0 creates a critically damped
-  /// spring, > 1.0 creates an overdamped spring and < 1.0 an underdamped one.
+  /// Creates a spring given the mass (m), stiffness (k), and damping ratio (ζ).
+  /// The damping ratio is especially useful trying to determing the type of
+  /// spring to create. A ratio of 1.0 creates a critically damped spring, > 1.0
+  /// creates an overdamped spring and < 1.0 an underdamped one.
   ///
-  /// See [mass] and [springConstant] for the units for those arguments. The
-  /// damping ratio is unitless.
+  /// See [mass] and [stiffness] for the units for those arguments. The damping
+  /// ratio is unitless.
   SpringDescription.withDampingRatio({
     this.mass,
-    this.springConstant,
+    this.stiffness,
     double ratio: 1.0
-  }) : damping = ratio * 2.0 * math.sqrt(mass * springConstant);
+  }) : damping = ratio * 2.0 * math.sqrt(mass * stiffness);
 
   /// The mass of the spring (m). The units are arbitrary, but all springs
   /// within a system should use the same mass units.
   final double mass;
 
-  /// The spring constant (k). The units of the spring constant are M/T², where
-  /// M is the mass unit used for the value of the [mass] property, and T is the
+  /// The stiffness of the spring (k). The units of stiffness are M/T², where M
```

Comment:
For clarity we should probably leave the first one here as it was ("The spring constant (k)"), that way someone looking for the term "spring constant" finds it.